packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -242,6 +236,7 @@ def get_nodes_from_response(response):
         nodes = []
         for node in response['nodes']:
             assert 'host_ip' in node, 'node response must have `host_ip` field. Got {}'.format(node)
+            # Fails with Windows
```

Comment:
Can you elaborate this?

Reply:
That's a left over. This was just for me to track it down. I'd like to have the CI setup first and then run it on this PR.